dash/frontend/src/app/core/breadcrumbInfo.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface BreadcrumbInfo {
+  place: number;  // location in the list
```

Comment:
It might be helpful to make the comments here into javadoc comments.

If property or function in a class/interface has a comment above that uses the multiline syntax with 2 stars after the first slash (/** ...  */) Then most IDEs will display that text as a tooltip when hovering over references to it elsewhere